keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -874,46 +858,29 @@ Think about adding additional work or introducing new steps in between
 
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
-Mutating API calls that opt-in to validation will be slower ([initial
-benchmarks](https://github.com/kubernetes/kubernetes/pull/104433#issuecomment-901398507)
-estimate ~20% slower 25-30% more memory consumption)
+Mutating API calls that opt-in to validation will be slower.
+Benchmarks of alpha changes indicate that performing validation results in
+~2-5% slower request latency and ~4-8% more memory usage.
```

Comment:
I'm glad to see that the numbers are actually smaller than initially provided (and the current ones actually better fit my intuition, which is great).